lib/driver.js

```diff
@@ -125,6 +126,17 @@ const AAB_EXT = '.aab';
 const SUPPORTED_EXTENSIONS = [APK_EXT, AAB_EXT];
 
 class EspressoDriver extends BaseDriver {
+
+  static newMethodMap = {
```

Comment:
I'd also put it into separate file as in android driver. More entries could be added there later

Reply:
ok, done